### PR TITLE
Table

### PR DESCRIPTION
**Pull request by Irirwanirira**
## This Pr consists of:

- [ ] Install SQL on my computer
- [ ] Create a database user 
- [ ] Create vet_clinic DATABASE
- [ ] Create an animal table
- [ ] Insert data into the table
- [ ] Practice queries in the terminal.
QUERY1

snap2

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,13 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (id, names, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (1,'Agumon', '2020-02-03', '0' , TRUE, '10.23' );
+
+INSERT INTO animals (id, names, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (2,' Gabumon', '2018-11-15', '3' , TRUE, '8' );
```
Comment (sja-thedude) [inline data.sql:7]: - [ ] Kindly make sure that the value for escape_attempts for Gabumon is 2 as per the requirements.

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,27 @@
-/*Queries that provide answers to the questions from all projects.*/
+/*Queries that provide answers to the questions FROM all projects.*/
+
+-- Find all animals whose names ends in "mon".
+
+SELECT * from animals where names LIKE '%mon';
+
+-- List the names of all animals born between 2016 and 2019.
+SELECT names from animals where date_of_birth  BETWEEN '2016-01-01' AND '2019-12-31';
+
```
Comment (sja-thedude) [inline queries.sql:9]: - [ ] Kindly make sure that you add screenshots for the one you have missed :
1) SELECT * from animals where names LIKE '%mon';
2) SELECT names from animals where date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Irirwanirira, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ Correct Gitflow

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (Irirwanirira) [reply data.sql:7]: Thank you for your time

Comment (Irirwanirira) [reply data.sql:7]: Thank you for your time

File: queries.sql (hunk 2 above)
Comment (Irirwanirira) [reply queries.sql:9]: @sja-thedude  Thank you for your time.

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @Irirwanirira 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Great job implementing the changes provided by the previous reviewer ✅
- Your project is professional ✅
- Well structured files ✅


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**